### PR TITLE
chore: Add update to `README.md` to explain undefined reference error

### PR DESCRIPTION
**Pull request by justinkambic**
## Summary

Resolves https://github.com/elastic/synthetics-recorder/issues/196.

Adds an update to the project's README that explains an error that can happen now that users are allowed to define and arrange their own step separators.

## Implementation details

Just a docs update.

## How to validate this change

Have a look at the example code and make sure it all makes sense. This code came directly from a real journey I ran, so it should be alright, but weirdness can happen.

Also find any typos, help eliminate passive voice, etc.

### CODE REVIEW

**Comment by justinkambic** [comment]
> My only suggestion (non blocking) is that maybe the Possible install issues section should go as a #### heading, higher up under ### Installation and Usage

Sure I'm fine adding that change.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,53 @@ Run the recorder app in dev mode.
 npm run dev
 ```
 
+### Troubleshooting
+
+#### Page object is not defined
+
+It is possible to define a series of steps that splits your recorded actions in a way that causes the required
+page object to be out of scope. You may be encountering this issue if, when testing or running your journey, you
```
Comment (vigneshshanmugam) [inline README.md:65]:
```suggestion
page object to be out of scope in the generated code. You may be encountering this issue if, when testing or running your journey, you
```

File: README.md [hunk 2]
```diff
@@ -57,6 +57,53 @@ Run the recorder app in dev mode.
...
+  ]);
+  await page2.click(':nth-match(a:has-text("babel-minify"), 3)');
+});
+step("Close page", async () => {
+  // referencing `page2`, which is now out of scope
+  await page2.close();
+});
+```
+
+By removing the second step divider in the Script Recorder UI, we will generate code that keeps the `page2` object
```
Comment (vigneshshanmugam) [inline README.md:90]:
```suggestion
By removing the step divider associated with `Close page` step in the Script Recorder UI, we will generate code that keeps the `page2` object
```

**Review by vigneshshanmugam — APPROVED** [review]
LGTM

**Review by kyungeunni — APPROVED** [review]
LGTM :shipit:

**Pull request merged**